src/infrastructure/implementations/crux-messenger.ts

```diff
@@ -117,3 +117,14 @@ export const cruxPaymentProtocol: IMessageSchema[] = [{
             .required().min(36).max(36),
     }),
 }];
+
+export const cruxConenctProtocol: IMessageSchema[] = [{
+    messageType: "CONNECT_REQUEST",
+    schema: Joi.object({
+        publicKey: Joi.string()
+            .required(),
```

Comment:
Fix length of public key. look at PAYMENT_REQUEST

Reply:
sure

Reply:
Max and Min length is fixed - just count length of any valid public key

Reply:
okay cool

Reply:
done